Lombiq.HelpfulLibraries/Libraries/Contents/ContentExtensions.cs

```diff
@@ -138,5 +138,31 @@ public static async Task SanitizeContentItemVersionsAsync(this IContent content,
             $"DisplayText: {content.ContentItem.DisplayText}, " +
             $"ID: {content.ContentItem.ContentItemId}, " +
             $"Version ID: {content.ContentItem.ContentItemVersionId}";
+
+        /// <summary>
+        /// Returns the most relevant date of the <paramref name="content"/>'s <see cref="ContentItem"/>.
+        /// </summary>
+        /// <returns>
+        /// <para>
+        /// The values are resolved in the following order if available. If all of them are <see langword="null"/> then
+        /// <see cref="DateTime.MinValue"/> is returned.
+        /// </para>
+        /// <list type="bullet">
+        ///     <item>
+        ///         <description><see cref="ContentItem.ModifiedUtc"/></description>
+        ///     </item>
+        ///     <item>
+        ///         <description><see cref="ContentItem.PublishedUtc"/></description>
+        ///     </item>
+        ///     <item>
+        ///         <description><see cref="ContentItem.CreatedUtc"/></description>
+        ///     </item>
+        /// </list>
+        /// </returns>
+        public static DateTime GetDateTime(this IContent content) =>

```

Comment:
Maybe `GetDateTimeUtc()` for clarity?

Reply:
Very sensible.

Reply:
Thanks for the drive-by review, Oliver :).

Reply:
It was my pleasure.